README.md

```diff
@@ -353,6 +353,11 @@ Once the above configurations have been made, the tool can be started with:
 ```
 docker compose up
 ```
+Note: This command requires your terminal session remain open. When you exit the shell, the Application Study Tool also exits.
```

Comment:
Thanks for submitting this. I think the better suggestion is have them do `docker compose up -d`. There are 3 spots where we mention `docker compose up` in the readme, if you can add a comment above each with the -d command that'll work - something like

```
docker compose up   # or `docker compose up -d` to run in the background
```